lib/ruboty/actions/jobcan.rb

```diff
@@ -55,6 +55,14 @@ def user_data
         brain_space[message.from_name] ||= {}
       end
 
+      def present_login_env?
+        !(
+          ENV["RUBOTY_JOBCAN_CLIENT_ID"].nil? &&
+          ENV["RUBOTY_JOBCAN_EMAIL"].nil?     &&
+          ENV["RUBOTY_JOBCAN_PASSWORD"].nil?
+        )
+      end
```

Comment:
`present_login_env?` returns true even if any of the env vars are unset. It should be

```ruby
def login_env_present?
  ENV["RUBOTY_JOBCAN_CLIENT_ID"] &&
  ENV["RUBOTY_JOBCAN_EMAIL"] &&
  ENV["RUBOTY_JOBCAN_PASSWORD"]
end
```